docs/source/gateway/index.rst

```diff
@@ -1,6 +1,971 @@
+If a given provider does not provide support for a parameter, a clear message will be returned when queried that explains the restrictions for the given provider and route type.
+
+- **temperature** (Supported by OpenAI, Anthropic, Cohere): This parameter controls the randomness of predictions by scaling the logits before applying softmax. A value closer to 0.0 makes the output more deterministic, while a value closer to 1.0 makes it more diverse. Default is 0.0.
+
+- **max_tokens** (Supported by OpenAI, Anthropic, Cohere): This parameter limits the length of the generated output by specifying a maximum token count. The range is from 1 to infinity, and by default, there is no limit (infinity). Some providers have a maximum value associated with this parameter that the AI Gateway will enforce to prevent a provider-generated exception.
+
+- **stop** (Supported by OpenAI, Anthropic, Cohere): This parameter specifies an array of strings, where each string is a token that indicates the end of a text generation. By default, this is empty.
+
+- **candidate_count** (Supported by OpenAI, Cohere): This parameter determines the number of alternative responses to generate. The range is from 1 to 5, and by default, it is set to 1.
+
+Alternate Query Parameters
+--------------------------
+These parameters are provider-specific and are not evaluated or checked by the MLflow AI Gateway for validity. If there are invalid values that are supplied for these, the provider's raw failure response will be returned.
+
+- **top_k** (Supported by Anthropic, Cohere): This parameter is used for "Top-K sampling", a text generation method where the K most likely next words are filtered and the probability mass is redistributed among only those K next words. This parameter is passed through directly to providers that support it, but is not a standard parameter of the AI Gateway API interfaces.
+
+- **logit_bias** (Supported by OpenAI, Cohere): This parameter allows you to add or subtract from the logits of the tokens before softmax, effectively making certain tokens more or less likely to appear. This parameter is passed through directly to providers that support it, but is not a standard parameter of the AI Gateway API interfaces.
+
+- **frequency_penalty** (Supported by OpenAI, Cohere): This parameter controls the penalty for using frequent tokens in the generated output. The higher the value, the less likely the model is to use common words. This parameter is passed through directly to providers that support it, but is not a standard parameter of the AI Gateway API interfaces.
+
+- **presence_penalty** (Supported by OpenAI, Cohere): This parameter controls the penalty for new tokens in the generated output. The higher the value, the more the model tends to repeat itself. This parameter is passed through directly to providers that support it, but is not a standard parameter of the AI Gateway API interfaces.
```

Comment:
No need to call these out explicitly, we can just say that users can pass any additional parameters they want except `stream=True/False`, such as `top_k` (Anthropic, Cohere), `logit_bias` (OpenAI, Cohere), and `presence_penalty` (OpenAI, Cohere)